tests/vmi_multus_test.go

```diff
@@ -797,11 +797,14 @@ var _ = Describe("[Serial]SRIOV", func() {
 			startVmi(vmi)
 			waitVmi(vmi)
 
-			vmi, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
-			Expect(err).NotTo(HaveOccurred())
-
-			interfaceName, err := getInterfaceNameByMAC(vmi, mac)
-			Expect(err).NotTo(HaveOccurred())
+			var interfaceName string
+			Eventually(func() error {
+				var err error
+				vmi, err = virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
+				Expect(err).NotTo(HaveOccurred())
+				interfaceName, err = getInterfaceNameByMAC(vmi, mac)
+				return err
+			}, 3*time.Minute, time.Second).Should(Succeed())
```

Comment:
Done, chosen 5 seconds in order the make test get the result faster.
I hope we wont see the missing report bug + 2 minutes interval causing flake, this is why i preferred larger time, smaller interval.
Anyway once the PR of reducing boot interval is merged we can play with those times, and if it will flake meanwhile we will know why.